<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPatient</class>
 <widget class="QWidget" name="AddPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="push_button_add_patient">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add patient</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pesel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PESEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_given_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Given name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_last_name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Last name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_of_birth">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date of birth</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_pesel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_given_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_last_name">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_sex">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="date_of_birth_edit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_header">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Insert patient data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="return_push_button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
